### PR TITLE
Small doc stutter typo

### PR DESCRIPTION
**Pull request by amoore877**
Minor typo fix

### CODE REVIEW

File: demo/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ cluster (cluster1) will host a greeter server workload. The second cluster
 (cluster2) will host a greeter client workload. Each cluster is a distinct
 SPIFFE trust domain.
 
-The greeter server and greeter client communicate with each over over TLS and
+The greeter server and greeter client communicate with each over TLS and
```
Comment (azdagron) [inline demo/README.md:11]: Hmm, should read "each other" :laughing: 
```suggestion
The greeter server and greeter client communicate with each other over TLS and
```

Comment (amoore877) [reply demo/README.md:11]: :)

**Review by azdagron — APPROVED** [review]
Thanks for noticing this, @amoore877 !

**Comment by azdagron** [comment]
I think the commits are missing DCO signoff.

**Comment by amoore877** [comment]
Well this was worth the try on directly editing files in github webpage

I think I'll just do the normal dev route unless we know how to add DCO through the browser

**Comment by azdagron** [comment]
When you edit the files on github, there is a place to put the commit header and optional body. You have to manually add the signoff to the body; it won't do it for you.

**Comment by amoore877** [comment]
Let's just start over with #23

**Pull request closed without merging**